.github/dependabot.yml

```
@@ -4,9 +4,13 @@ updates:
     directory: "/dashboard"  
     schedule:
       interval: "weekly"
+      time: "23:00"
+      timezone: Europe/London
       labels:
-      - "npm dependencies"
+      - "dependencies"
+    open-pull-requests-limit: 10
```

```suggestion
    open-pull-requests-limit: 5
```

10 is bit too high @Azanul wdyt?

---

Nvm I think 10 sounds reasonable for now, we can change those when needed later on

---

I want to tell you why 10? because we dependent on more than 10 node dependencies.